### PR TITLE
Taxon Display component created, properly passed data from filter and filter row

### PR DESCRIPTION
**Pull request by GraysonQuartech**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import FilterRow from "./components/FilterRow";
+import TaxonDisplay from "./components/TaxonDisplay";
 
 const App = () => {
```
Comment (MacQSL) [inline src/App.tsx:5]: return type

File: src/components/Filter.tsx [hunk 2]
```diff
@@ -23,17 +41,24 @@ interface filterProps {
...
+    props.onSelectedChange(selectedTaxon);
   };
 
+  //RETURN ELEMENT HERE
   return (
-    <Box>
+    <Box className={classes.container}>
       <FormControl fullWidth>
         <InputLabel id="select-label">{props.classificationLevel}</InputLabel>
         <Select
```
Comment (MacQSL) [inline src/components/Filter.tsx:60]: use mui autocomplete

**Review by MacQSL — COMMENTED** [review]
some small changes, looking good so far

File: src/components/Filter.tsx [hunk 3]
```diff
@@ -3,6 +3,20 @@ import * as React from "react";
...
+/*
+ * STYLE definitions for useStyles hook
+ */
+const useStyles = makeStyles({
+  container: {
+    padding: "0.5rem",
+    display: "block",
+  },
+  selectbox: {
+    backgroundColor: "white",
```
[the remark this answers is not in the document]
Comment (GraysonQuartech) [reply src/components/Filter.tsx]: will handle theme stuff in next PR

File: src/components/Filter.tsx [hunk 4]
```diff
@@ -3,6 +3,20 @@ import * as React from "react";
...
 import Select, { SelectChangeEvent } from "@mui/material/Select";
 import { Box, FormControl, InputLabel, MenuItem } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+
+/*
+ * STYLE definitions for useStyles hook
+ */
+const useStyles = makeStyles({
+  container: {
+    padding: "0.5rem",
```
[the remark this answers is not in the document]
Comment (GraysonQuartech) [reply src/components/Filter.tsx]: will handle theme stuff in next PR

**Pull request merged**